DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Behavioiur : MonoBehaviour {
+    private GameObject player;
+    private Rigidbody2D rb;
+    public int thrust;
+    private CircleCollider2D circle;
+
+    // Use this for initialization
+    void Start () {
+        player = GameObject.FindGameObjectWithTag("Player");
+        rb = GetComponent<Rigidbody2D>();
+        circle = GetComponent<CircleCollider2D>();
+        thrust = 100;
```

Comment:
if i remove line 16, then you can put whatever value you want in the inspector. Want me to take it out?